src/WebLinks/GitHubWebProvider.cs

```diff
@@ -8,9 +8,15 @@ internal class GitHubWebProvider : IWebProvider
 
         public bool CanHandle(string repositoryUrl)
         {
-            var uri = new Uri(repositoryUrl);
-
-            return uri.Host.Equals("github.com");
+            try
+            {
+                var uri = new Uri(repositoryUrl);
+                return uri.Host.Equals("github.com");
+            }
+            catch (UriFormatException)
+            {
+                return false;
```

Comment:
Good point.. SSH remotes for GitHub would also be broken. I'll create an issue for this.
Thanks! :+1:

Reply:
New issue for reference: https://github.com/mjcheetham/vs-weblinks/issues/13